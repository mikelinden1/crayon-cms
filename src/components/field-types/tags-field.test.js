import TagsField from './tags-field';

const props = {
    name: 'test',
    value: ['val1', 'val2', 'val3'],
    onChange: () => {},
    disabled: false
};

it('TagsField matches snapshot', () => {
    const wrapper = shallow(<TagsField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
