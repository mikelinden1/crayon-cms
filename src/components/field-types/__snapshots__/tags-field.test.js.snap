// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagsField matches snapshot 1`] = `
<TagsInput
  addKeys={
    Array [
      9,
      13,
    ]
  }
  addOnBlur={false}
  addOnPaste={false}
  className="react-tagsinput"
  disabled={false}
  focusedClassName="react-tagsinput--focused"
  inputProps={Object {}}
  maxTags={-1}
  onChange={[Function]}
  onlyUnique={false}
  pasteSplit={[Function]}
  preventSubmit={true}
  removeKeys={
    Array [
      8,
    ]
  }
  renderInput={[Function]}
  renderLayout={[Function]}
  renderTag={[Function]}
  tagDisplayProp={null}
  tagProps={
    Object {
      "className": "react-tagsinput-tag",
      "classNameRemove": "react-tagsinput-remove",
    }
  }
  validationRegex={/\\.\\*/}
  value={
    Array [
      "val1",
      "val2",
      "val3",
    ]
  }
/>
`;
